src/components/__tests__/VideoPlayer.spec.tsx

```diff
@@ -0,0 +1,165 @@
+import { shallow } from "enzyme";
+import * as React from "react";
+
+import { VideoPlayer } from "../VideoPlayer";
+import { Youtube } from "../Youtube";
+import { Vimeo } from "../Vimeo";
+import { Dailymotion } from "../Dailymotion";
+import { Html5Player } from "../Html5Player";
+import { PlayerError } from "../PlayerError";
+import { fixHeightWithRatio, getRatio, validateUrl } from "../../utils/Utils";
+
+describe("Video Player", () => {
+    it("Renders the structure of youtube tags", () => {
+        const player = shallow(<VideoPlayer
+            url="http://youtube.com/video/123456"
+            staticUrl=""
+            poster=""
+            staticPoster=""
+            autoStart={false}
+            showControls={false}
+            loop={false}
+            muted={false}
+            aspectRatio={false}
+        />);
+
+        expect(player).toBeElement(
+            <Youtube
+                url="http://youtube.com/video/123456"
+                showControls={false}
+                autoPlay={false}
+                muted={false}
+                loop={false}
+                aspectRatio={false}
+            />
+        );
+    });
+
```

Comment:
Add test case
`it("Renders the structure with custom style")`
`it("Renders the structure with custom class")`